qiskit/ignis/verification/accreditation/circuits.py

```diff
@@ -14,410 +14,131 @@
-    # other qubits (qubits can be entanged one time per band)
-    unavailable_qubits = []
-
-    # Keep track of current band
-    current_band_no = 0
-
-    # Loops over all gates in the circuit. An extra element is added so the
-    # last band is closed at the end
-    for gate in target_circuit.data + ['END STRING']:
-
-        # Checks for special cases that need to be handled differently
-        gate_qubits = gate[1]
-        last_element = (gate == 'END STRING')
-        is_measure = ((len(gate_qubits) == len(gate[2]))
-                      and not gate == 'END STRING')
-
-        # Records the position of the last cz gate
-        if cz_gate:
-            last_cz = cz_gate[-1][0]
-        else:
-            last_cz = -1
-
-        # Makes sure measurements are ignored
-        if not is_measure:
-            circuit_end_band = last_element\
-                and ((last_cz == current_band_no)
-                     or (single_qubit_gates != [[] for _ in circuit_qubits]))
-
-            # If a new band is required, converts the current band's single
-            # qubit gates to Euler angles and prepares for the next band
-            if((not set(gate_qubits).isdisjoint(set(unavailable_qubits)))
-               or circuit_end_band):
-                band_gates_angles = []
-                u3_gates_temp = []
-                for qubit in single_qubit_gates:
-                    matrix = np.array([[1, 0], [0, 1]])
-                    for gate_index in qubit[::-1]:
-                        matrix = np.matmul(matrix, gate_index[0].to_matrix())
-                    band_gates_angles.append(euler_angles_1q(matrix))
-                    u3_gates_temp.append(band_gates_angles[0])
-                    band_gates_angles = []
-                gates_target.append(u3_gates_temp)
-                current_band_no += 1
-                unavailable_qubits = []
-                single_qubit_gates = [[] for _ in circuit_qubits]
-            if last_element:
-                break
-
-            # Adds single gates' gate object to the array corresponding to
-            # their qubit in single_qubit_gates
-            if len(gate_qubits) == 1:
-                single_qubit_gates[gate_qubits[0].index].append(gate)
-            # Records the location of 2 qubit gates
-            else:
-                cz_gate.append([current_band_no, gate_qubits[0].index,
-                                gate_qubits[1].index])
-                unavailable_qubits += gate_qubits
-
-    # Adds a band of identity gates if circuit ends with cz gates
-    if last_cz == current_band_no - 1:
-        last_band = []
-        for qubit in circuit_qubits:
-            last_band.append((0.0, 0.0, 0.0))
-        gates_target.append(last_band)
+        Load target circuit in to class,
+        parse into layers
+            Args:
+                targetcirc (circuit): a qiskit circuit to accredit
+                coupling_map (list): some particular device topology
```

Comment:
two_qubit_gate is missing here. See above comment for why this wasn't flagged by linter